smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolServiceClient.kt

```diff
@@ -30,6 +30,7 @@ open class HttpProtocolServiceClient(
             ClientRuntimeTypes.Core.Client,
         ) {
             writer.write("public static let clientName = \$S", serviceSymbol.name)
+            writer.write("public static let version = \$S", ctx.settings.moduleVersion)
```

Comment:
The client gains this version property, which is used over in the accompanying AWS SDK PR to set the client version in the user agent.